test/e2e/operator_groups_e2e_test.go

```diff
@@ -523,6 +523,28 @@ func TestOperatorGroupRoleAggregation(t *testing.T) {
 		return res.Status.Allowed, nil
 	})
 	require.NoError(t, err)
+	err = wait.Poll(pollInterval, pollDuration, func() (bool, error) {
```

Comment:
@ecordell how is this related to grpc bump?

Reply:
Thanks for catching this - must've been a weird rebase issue.